graphql/mutation_resolver.go

```diff
@@ -1265,8 +1265,23 @@ func (r *mutationResolver) UpdateUserSettings(ctx context.Context, userSettings
 
 // RemoveItemFromCommitQueue is the resolver for the removeItemFromCommitQueue field.
 func (r *mutationResolver) RemoveItemFromCommitQueue(ctx context.Context, commitQueueID string, issue string) (*string, error) {
-	username := gimlet.GetUser(ctx).DisplayName()
-	result, err := data.FindAndRemoveCommitQueueItem(ctx, commitQueueID, issue, username, fmt.Sprintf("removed by user '%s'", username))
+	user := mustHaveUser(ctx)
+
+	project, err := data.FindProjectById(commitQueueID, false, false)
```

Comment:
includeRepo should be true, since a project could inherit commit queue settings from the repo (might be worth adding a test for this)